src/openrct2/ride/Vehicle.cpp

```diff
@@ -7322,92 +7334,73 @@ void Vehicle::UpdateAdditionalAnimation()
                             }();
                             int32_t directionIndex = sprite_direction >> 1;
                             auto offset = SteamParticleOffsets[typeIndex][directionIndex];
-                            SteamParticle::Create({ x + offset.x, y + offset.y, z + offset.z });
+                            SteamParticle::Create(CoordsXYZ(
+                                x + offset.x * vehicleEntry->SteamEffect.Longitudinal / SteamEffectTranslationCoefficient,
+                                y + offset.y * vehicleEntry->SteamEffect.Longitudinal / SteamEffectTranslationCoefficient,
+                                z + offset.z * vehicleEntry->SteamEffect.Vertical / SteamEffectTranslationCoefficient));
```

Comment:
"Coefficient" doesn't seem right when it's used for division.

Reply:
What would you call it then?

Reply:
@Broxzier

Reply:
I think its fine. I've used factor in some places but coefficient / factor they are all fine.